sqrl-testing/sqrl-integration-tests/src/test/java/com/datasqrl/FullUseCasesIT.java

```diff
@@ -334,64 +336,70 @@ void printUseCaseNumbers(UseCaseTestParameter param) {
   @Test
   @Disabled
   public void runTestCaseByName() {
```

Comment:
I would prefer having a new test just for external projects....

Reply:
That's where i started as well, and ended up with some duplicates so merged them, but lets see, i'll give it another spin and abstract the common parts